docs/root/manual/getting-started/run-signer-node.md

```diff
@@ -454,10 +581,84 @@ sudo iptables -L -v
 sudo service netfilter-persistent save
 ```
 
+:::tip
+
+Here is an example command created with the example set for `release-preprod`:
+
+```bash
+sudo iptables -A INPUT -s 192.168.1.75 -p tcp --dport 3128 -j ACCEPT
+sudo iptables -L -v
+sudo service netfilter-persistent save
+```
+
+:::
+
 ## Verify the Mithril Signer deployment
 
 :::tip
 There is a delay of `2` epochs between the registration of the signer node and its ability to generate individual signatures. This delay is further explained in the [Mithril certificate chain in depth](https://mithril.network/doc/mithril/mithril-protocol/certificates) documentation.
 
 Once this delay has passed, you should be able to observe your `PoolId` listed in some of the certificates accessible on the [`Mithril Explorer`](https://mithril.network/explorer).
 :::
+
+### Verify your signer is registered
+
+After installing the Mithril Signer, you can verify that your node is registered by checking your Mithril Signer node logs.  
+
+First, download the script in the desired directory:
+
+```bash
+wget "https://raw.githubusercontent.com/input-output-hk/mithril/main/docs/scripts/verify_signer_registration.sh"
+```
+
+Make the script executable:
+
+```bash
+chmod +x verify_signer_registration.sh
+```
+
+Finally, execute the script:
+```bash
+SIGNER_LOGS_PATH=**YOUR_SIGNER_LOGS_PATH** ./verify_signer_registration.sh
+```
+
+:::tip
+
+Here is an example command:
+
+```bash
+SIGNER_LOGS_PATH=/var/log/syslog ./verify_signer_registration.sh
+```
+
+:::
+
+### Verify your signer contributes with individual signatures
```

Comment:
Can you add an examples of the information displayed when contributing and not contributing at the end of the section?

Reply:
**_Result in the explorer :_**

![image](https://github.com/input-output-hk/mithril/assets/135982616/54ad2616-845f-44e6-a3df-c219b067c022)

For information, the hash mentioned at the end of the url is a fake hash.
```bash
>> Congrats, you have signed this certificate: https://aggregator.release-preprod.api.mithril.network/aggregator/certificate/el3p289b03a223244285b2ls10839846ae7a69f1e8362824a383f376f93f723f !
```

Is this ok for you ?

Reply:
Great!